### DISCUSSION TITLE
Jeet para grid system

### DISCUSSION THREAD

**Discussion by matheusmpessoa** [frontendbr/forum#604] [Perguntas e Respostas]
Ao pesquisar pelo google e pelo próprio fórum, vi que muitos utilizam o [Jeet](http://jeet.gs/) para grid system.
Alguém aqui já usou? É recomendável? Alguém indica? Já tiveram alguma dor de cabeça com essa ferramenta?

Obrigado!!

**Comment by nandomoreirame** [comment]
Eu já usei ele com SASS... eles deram uma parada agora voltaram a ativa.
um atenção: dependendo de como vc usar ele o seu código final pode ficar um pouco "sujo", é bom medir isso desde o começo :+1:

**Comment by Cleysonlb** [comment]
@matheusmpessoa eu uso e indico, é um sistema de grid bem completo e não é travado igual bootstrap, é bem simples de usar! 👍

**Comment by marcusvcribeiro** [comment]
Usei bastante tempo, inclusive indico, só que eu parei quando fiquei
sabendo que nao ia ter mais continuidade, agora se voltou a ter, ai é uma
otima ideia, porque voltarei a usar,


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Livre
de vírus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>.
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2017-06-26 18:01 GMT-03:00 Cleyson Leal Braga <notifications@github.com>:

**Comment by felipefialho** [comment]
Usei por muito tempo, mas hoje em dia faz mais sentido usar o [Lost](https://github.com/peterramsing/lost) como Grid System, muito mais moderno e dos mesmos criadores.

**Comment by matheusmpessoa** [comment]
@LFeh mas para usar em conjunto do Sass acredito que seja melhor o Jeet, visto que o Lost utiliza PostCSS, correto?

**Comment by felipefialho** [comment]
@matheusmpessoa Não

PostCSS significa que é pós compilado, ou seja, funciona com qualquer pré-processador e até mesmo com CSS puro :)
